contracts/src/v0.7/SafeMath96.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.0;
+
+/**
+ * @dev Wrappers over Solidity's arithmetic operations with added overflow
+ * checks.
+ *
+ * Arithmetic operations in Solidity wrap on overflow. This can easily result
+ * in bugs, because programmers usually assume that an overflow raises an
+ * error, which is the standard behavior in high level programming languages.
+ * `SafeMath` restores this intuition by reverting the transaction when an
+ * operation overflows.
+ *
+ * Using this library instead of the unchecked operations eliminates an entire
+ * class of bugs, so it's recommended to use it always.
+ *
+ * This library is a version of Open Zeppelin's SafeMath, modified to support
+ * unsigned 96 bit integers.
+ */
+library SafeMath96 {
```

Comment:
Let's move this into `vendor/` with the other SafeMath contracts

Reply:
done